src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.ts

```diff
@@ -62,20 +179,32 @@ export async function getIndices(
     return [];
   }
 
-  const query = showAllIndices ? { expand_wildcards: 'all' } : undefined;
+  const promiseResolve = getIndicesViaResolve({
+    http,
+    getIndexTags,
+    pattern,
+    showAllIndices,
+  }).catch(() => []);
+  requests.push(promiseResolve);
+
+  if (isCCS) {
+    // CCS supports ±1 major version. We won't be able to expect resolve endpoint to exist until v9
+    const promiseSearch = getIndicesViaSearch({
+      getIndexTags,
+      pattern,
+      searchClient,
+      showAllIndices,
+    }).catch(() => []);
```

Comment:
Yes, we're catching errors without notifying the user. It seems that it would be better to notify the user but that would be more work and we haven't had any complaints. Cross cluster calls are more likely to fail and we did get complaints about slow clusters that were preventing the creation of index patterns.